components/pci/component.go

```diff
@@ -256,16 +238,21 @@ func (d *Data) getStates() ([]components.State, error) {
 	return []components.State{state}, nil
 }
 
-func (d *Data) createEvent(time time.Time) *components.Event {
-	uuids := d.listACSEnabledDeviceUUIDs()
-	if len(uuids) == 0 {
+func (d *Data) listACSEnabledDeviceUUIDs() []string {
```

Comment:
It seems that we can make it as a freedom function, not necessary to make it as a method of Data.